HelpSource/Classes/MouseButton.schelp

```diff
@@ -35,5 +35,17 @@ code::
 { SinOsc.ar(MouseButton.kr(400, 440, 0.1), 0, 0.1) }.play;
 { SinOsc.ar(MouseButton.kr(400, 740, 2), 0, 0.1) }.play;
 
+(
+SynthDef( \mousexyb, { |out=0|
+	var mousex, mousey, mousebutton;
+	mousex = MouseX.kr( 500, 1000 ); // this will determine the frequency of the sound (minimum value, maximum value, warp, lag)
```

Comment:
If you're going to write what the arguments are in the comments, why not write the argument names?:

```
MouseX.kr(minval: 500, maxval: 1000)
```

Same goes for "MouseButton" below